userguide/tasks.rst

```diff
@@ -168,7 +168,11 @@ shows the configuration options for Cloud Syncs.
    | Enabled                | checkbox            | Unset to temporarily disable this Cloud Sync.                                                          |
    |                        |                     |                                                                                                        |
    +------------------------+---------------------+--------------------------------------------------------------------------------------------------------+
-
+   
+   
+.. note:: The time chosen for the Cloud Sync refers to the time that
```

Comment:
This is kind of redundant and not very clear.

`` The time selected is when the Cloud Sync task is allowed to begin. See :ref:`Cron Jobs` for setting the time when Cloud Syncs are attempted. ``

Please verify that this is correct (Cloud Sync times are set as cron jobs, not in Cloud Sync).

Reply:
I spoke with William. The Cloud Sync time chosen is when the task will run. It has nothing to do with Cron Jobs in the UI.